### PR TITLE
Fix non working example

### PR DESCRIPTION
**Pull request by e-minguez**
The nginx example doesn't work as it requires root permissions. Using the openshift/hello-openshift is also better for self promoting openshift :)

### CODE REVIEW

**Review by cooktheryan — APPROVED** [review]

File: dev_guide/deployments/kubernetes_deployments.adoc [hunk 1]
```diff
@@ -32,22 +32,22 @@ xref:../../architecture/core_concepts/deployments.adoc#replication-controllers[r
 For example, this definition of a Kubernetes deployment creates a replica set to
```
Comment (ahardin-rh) [inline dev_guide/deployments/kubernetes_deployments.adoc]: Looks like the wording here needs to be updated to match the example below.

**Comment by ahardin-rh** [comment]
@kargakis PTAL

**Comment by ahardin-rh** [comment]
@kargakis I still see a reference to nginx just before the example. Once that is updated, I will merge. Thanks!

**Comment by ahardin-rh** [comment]
To get this moving along, I will do a follow-up edit in a separate PR

**Pull request merged**